speech/google/cloud/speech/sample.py

```diff
@@ -112,3 +117,180 @@ def encoding(self):
         :returns: String value of Encoding type.
         """
         return self._encoding
+
+    def async_recognize(self, language_code=None, max_alternatives=None,
+                        profanity_filter=None, speech_context=None):
+        """Asychronous Recognize request to Google Speech API.
```

Comment:
1. A ll just copy-pasta or do i need to read?
1.  Why not just put the remaining four args (`language_code`, etc.) into the constructor? Or if not in the constructor make a "query" class (or similar) since both sync and async have the same args

    ```python
    query = sample.query(
        language_code='en-US', max_alternatives=4)

    # async is a keyword in Python3.5 so might need a diff name
    query.async()
    query.sync()
    ```